cypress/e2e/awx/administration/topology-view.cy.ts

```diff
@@ -125,7 +134,7 @@ describe('Topology view', () => {
       });
   });
 
-  it('should render download icon for instance that has an install bundle', () => {
+  it.skip('should render download icon for instance that has an install bundle', () => {
```

Comment:
I see that you have skipped this test- It looks like it would be a better component test. Perhaps it can be moved to the component test file for this feature, and deleted from this file?